packages/react/src/components/TextField/TextField.tsx

```diff
@@ -16,29 +16,108 @@
  * under the License.
  */
 
-import {InputLabel} from '@mui/material';
+import InputAdornment from '@mui/material/InputAdornment';
 import MuiTextField, {TextFieldProps as MuiTextFieldProps} from '@mui/material/TextField';
+import {DoubleCircleIcon, VisibilityIcon, VisibilityOffIcon} from '@oxygen-ui/react-icons';
 import clsx from 'clsx';
-import {FC, ReactElement} from 'react';
+import {FC, MouseEvent, ReactElement, ReactNode, useState} from 'react';
 import {WithWrapperProps} from '../../models';
 import {composeComponentDisplayName} from '../../utils';
+import IconButton from '../IconButton';
+import InputLabel from '../InputLabel';
+import List from '../List';
+import ListItem from '../ListItem';
+import ListItemIcon from '../ListItemIcon';
+import ListItemText from '../ListItemText';
+import Tooltip from '../Tooltip';
 import './text-field.scss';
 
-export type TextFieldProps = MuiTextFieldProps;
+export type TextFieldProps = {
```

Comment:
Lets use an interface.

Reply:
Cannot extend this with an interface as `MuiTextFieldProps` is a union type. This was the solution according to the thread, `https://github.com/mui/material-ui/issues/32633`.

Reply:
Cool 👍